### PR TITLE
Revision on sections 2.4 and 5.2 according to Dom's comments

### PR DESCRIPTION
**Pull request by panll**
Sections 2.4 and 5.2 are revised according to Dom's comments.
https://github.com/ufs-community/ufs-mrweather-app/issues/207

### CODE REVIEW

File: docs/UsersGuide/source/introduction.rst [hunk 1]
```diff
@@ -183,7 +183,7 @@ the distributed documentation, summarized here for ease of use.
    +----------------------------+---------------------------------------------------------------------------------+
    | **Documentation**          | **Location**                                                                    |
    +============================+=================================================================================+
-   | MR Weather App v1.1    | https://ufs-mrweather-app.readthedocs.io/en/ufs-v1.1.0                          |
+   | MR Weather App v1.1        | https://ufs-mrweather-app.readthedocs.io/en/ufs-v1.1.0                          |
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/introduction.rst:186]: It seems this table is not showing in the HTML. Julie had fixed this in a recent PR. Please check.

**Review by ligiabernardet — CHANGES_REQUESTED** [review]

File: docs/UsersGuide/source/introduction.rst (hunk 1 above)
Comment (panll) [reply docs/UsersGuide/source/introduction.rst:186]: Julie's change is included. I can see the table 1.1.

**Review by climbfuji — COMMENTED** [review]
THanks for making the updates w.r.t. the `--machine` argument. Those look good. I am not sure what is going on with the changes in the table - did you pull in the latest version after we merged Julie's changes?

**Comment by panll** [comment]
I manually changed.

**Comment by climbfuji** [comment]
> I manually changed.

Thanks. I'll wait for @ligiabernardet's approval.

**Comment by ligiabernardet** [comment]
@panll When I look at the HTML at RTD from your fork/branch (https://123456789.readthedocs.io/en/latest/introduction.html#how-to-use-this-document), I do not see the table. Do you?

**Review by ligiabernardet — APPROVED** [review]
OK, I see the table is there. Somehow I could not see it before. Tks

**Pull request merged**